src/GSoft.Extensions.MediatR.Tests/GSoft.Extensions.MediatR.Tests.csproj

```diff
@@ -8,6 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\GSoft.Extensions.MediatR.Analyzers\GSoft.Extensions.MediatR.Analyzers.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
```

Comment:
This is how to reference a project only to consume it's analyzers and without publishing its DLL to the output directory